instruments/qubitekk/mc1.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Provides support for the Qubitekk MC1 Motor Controller.
+
+MC1 Class originally contributed by Catherine Holloway.
+"""
+
+# IMPORTS #####################################################################
+from __future__ import absolute_import
+from enum import Enum
+from numpy import arange
```

Comment:
I'd rather us not rely on `numpy` more than we need to as my eventual goal will be to remove `numpy` as a dependency